creme/creme_config/templates/creme_config/forms/widgets/buttonmenu-editor.html

```diff
@@ -0,0 +1,23 @@
+{% load creme_core_tags creme_widgets i18n %}
+
+<div class="buttonmenu-edit-widget" id="{{widget.attrs.id}}">
+  <div class="widget-available buttons-list instance-buttons">
+    <div class="buttons-list-header">{% trans "Available buttons" %}</div>
+    <div class="widget-container"></div>
+  </div>
+  <div class="widget-selected buttons-list instance-buttons">
+    <div class="buttons-list-header">{% trans "Selected buttons" %}</div>
+    <div class="widget-container"></div>
+  </div>
+</div>
+
+{{ widget.choices|json_script:"buttons-widget-choices" }}
+
+<script type="text/javascript">
```

Comment:
Hum... you can create a creme widget instead off initialize it in scripts. But if is the widget is used once, why not.